scanned.py

```diff
@@ -3,8 +3,21 @@
 # The images must temporarily be stored in "./out/imgs/", and deleted after being processed.
 
 # Imports
-# TODO
+import pytesseract  
 
 # The extraction function
 def extract(filename):
-	... # TODO
\ No newline at end of file
+	# TODO
+	# Open the file
+	# Extract text
+	# Save text to file
+	# Close the file
+	# Open the file
+	with open(filename, 'rb') as f:
+		# Extract text
+		pdfReader=pytesseract.image_to_string(f)
+		# Save text to file
+		with open('out/output.txt', 'w') as f:
+			f.write(pdfReader)
+		# Close the file
+		f.close()

```

Comment:
This is wrong, Yaswanth. Refer: https://github.com/Manvendra2000/Reader/blob/master/reader.py#L6-L111